src/syngen/ml/worker/worker.py

```diff
@@ -349,6 +390,21 @@ def __infer_tables(self, tables: List, config_of_tables: Dict):
                 message=f"Infer process of the table - {table} was completed"
             )
 
+        MlflowTracker().start_run(run_name='integral_metrics', tags={'process':'bottleneck'})
```

Comment:
It seems that it might be a separate method of the Worker, because this code is duplicated in the method '__train_tables'